src/ImageSharp/Formats/PixelTypeInfo.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace SixLabors.ImageSharp.Formats
+{
+    /// <summary>
+    /// Stores information about pixel.
```

Comment:
We need a better description than this.

I don't like the name. `RawPixelTypeInfo` perhaps? Naming is hard!

Reply:
@JimBobSquarePants I proposed this class and naming in order to make it extendable in a way like this:

```C#
public class PixelTypeInfo
{
    public System.Type PixelType { get; }
    public int BitsPerPixel { get; }

    internal PixelTypeInfo(int bitsPerPixel, System.Type pixelType = null)
    {
        this.BitsPerPixel = bitsPerPixel;
    }

    public static PixelTypeInfo Create<TPixel>() where TPixel 
        :  IPixel<TPixel> => 
        new PixelTypeInfo(Unsafe.SizeOf<TPixel>(), typeof(TPixel));
}
```

This will enable integrating it deeper with our pixel type infrastructure, and provide non-generic pixel-type related API-s. As a bridge between native (image format/stream bound) pixel type data and our internal pixel types, this class doesn't necessarily represent "raw"/native pixel type information.

Reply:
See this https://github.com/SixLabors/ImageSharp/pull/292#discussion_r162216541.